libs/wingc/src/visit.rs

```diff
@@ -124,6 +124,19 @@ where
 			v.visit_scope(statements);
 		}
 		StmtKind::Break | StmtKind::Continue => {}
+		StmtKind::IfLet {
+			condition,
+			statements,
+			var_name,
+			else_statements,
+		} => {
+			v.visit_expr(condition);
+			v.visit_scope(statements);
+			v.visit_symbol(var_name);
```

Comment:
Reorder (need to visit AST in syntax order - I believe some code in the LSP relies on this property)
```suggestion
			v.visit_symbol(var_name);
			v.visit_expr(condition);
			v.visit_scope(statements);
```